include/wx/calctrl.h

```diff
@@ -50,7 +47,10 @@ enum
     wxCAL_SHOW_SURROUNDING_WEEKS     = 0x0020,
 
     // show week numbers on the left side of the calendar.
-    wxCAL_SHOW_WEEK_NUMBERS          = 0x0040
+    wxCAL_SHOW_WEEK_NUMBERS          = 0x0040,
+
+	// show Sunday as the first day of the week (default)
+    wxCAL_SUNDAY_FIRST               = 0x0080
```

Comment:
I don't remember what was the initial rationale for moving the enum elements around, but it doesn't seem necessary any more, so I'd like to avoid modifying this at all to avoid spurious diffs.

Reply:
The latest rationale was keeping the values in ascending order; I have now canceled moving the elements around, and just changed the value of `wxCAL_SUNDAY_FIRST`. If anyone adds a new element to the enum later, they'll need to notice that 0x80 is already reserved.